libmscore/key.cpp

```diff
@@ -188,7 +188,7 @@ AccidentalVal AccidentalState::accidentalVal(int line, bool &error) const
             error = true;
             return AccidentalVal::NATURAL;
             }
-      return AccidentalVal((state[line] & 0x0f) - 2);
+      return AccidentalVal((state[line] & 0x0f) - + int(AccidentalVal::MIN));
```

Comment:
There is an extraneous `-` on this line. This is the cause of some and perhaps all of the mtest failures. It should be
```
      return AccidentalVal((state[line] & 0x0f) + int(AccidentalVal::MIN));
```

Reply:
Outch,  copy&paste error, thanks!